packages/typespec-test/test/openai_modular/generated/typespec-ts/src/api/index.ts

```diff
@@ -1,25 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export {
-  Embeddings,
-  EmbeddingItem,
-  EmbeddingsUsage,
-  Completions,
-  Choice,
-  CompletionsLogProbabilityModel,
-  CompletionsFinishReason,
-  CompletionsUsage,
-  ChatMessage,
-  ChatRole,
-  ChatCompletions,
-  ChatChoice,
-  ImageOperationResponse,
-  ImageResponse,
-  ImageLocation,
-  State,
-  ImageSize,
-} from "./models.js";
```

Comment:
This is the part where I mentioned to remove the export for models in api layer index.ts